### PR TITLE
Add steps to declare new fields in CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by poirierlouis**
Fix typos.
Provide insights and tips on how generated classes can be extended with reverse engineered members.

Let me know if there are mistakes / steps I missed / ...

### CODE REVIEW

**Review by psiberx — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -53,7 +53,7 @@ The project is using [C++20](https://en.cppreference.com/w/cpp/20), any C++20 fe
 
 ### Macros
 
-* Marcos should named with uppercase letters.
+* Marcos should be named with uppercase letters.
```
[the remark this answers is not in the document]
Comment (poirierlouis) [reply CONTRIBUTING.md]: Thanks!

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -72,6 +72,99 @@ and indicate which issue is fixed.
...
++RED4EXT_ASSERT_OFFSET(BaseObject, isVehicleOnStateLocked, 0x6D2);
+} // namespace vehicle
+using vehicleBaseObject = vehicle::BaseObject;
+using VehicleObject = vehicle::BaseObject;
+} // namespace RED4ext
+```
+
+### Generate stub
+
+1. Build the library and install it in `<game>/red4ext/plugins/`.
```
Comment (WopsS) [inline CONTRIBUTING.md:150]: I would say something like `Build the RTTIDumper` with the modified SDK and install ...`

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -72,6 +72,99 @@ and indicate which issue is fixed.
...
+2. Run the game, wait, close the game and go in `<game>/bin/x64/dumps/cpp/Scripting/`.
+3. Copy the folder `Natives` in the folder of RED4ext.SDK to replace its content with the new version.
```
Comment (WopsS) [inline CONTRIBUTING.md:152]: First I would say `Delete the old folder and the copy the new one`

**Review by WopsS — CHANGES_REQUESTED** [review]
Thanks for the changes <3

File: CONTRIBUTING.md (hunk 2 above)
Comment (poirierlouis) [reply CONTRIBUTING.md]: Should be good.

File: CONTRIBUTING.md (hunk 3 above)
Comment (poirierlouis) [reply CONTRIBUTING.md]: Done, with an extra bullet point.

**Comment by poirierlouis** [comment]
I added a step to remove generated comments.

**Review by WopsS — APPROVED** [review]

**Review by psiberx — APPROVED** [review]

**Pull request merged**